catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/platform/transformer/transformers/sierra/SierraProduction.scala

```diff
@@ -150,6 +147,42 @@ trait SierraProduction {
         )
       }
 
+  /** Populate the production data if both 260 and 264 are present.
+    *
+    * In general, this is a cataloguing error, but sometimes we can do
+    * something more sensible depending on if/how they're duplicated.
+    */
+  private def getProductionFromBothFields(marc260fields: List[VarField],
+                                          marc264fields: List[VarField]) = {
+
+    // We've seen cases where the 264 field only has the following subfields:
+    //
+    //      [('tag', 'c'), ('content', '©2012')]
+    //
+    // or similar, and the 260 field is populated.  In that case, we can
+    // discard the 264 and just use the 260 fields.
+    if (marc264fields.length == 1 &&
+        marc264fields.head.subfields.length == 1 &&
+        marc264fields.head.subfields.head.tag == "c" &&
+        marc264fields.head.subfields.head.content.matches("^©\\d+$")) {
```

Comment:
Tighten this regex – it should be 4 digits.

Reply:
👍